### DISCUSSION TITLE
Abandon pull requests to ignore updates?

### DISCUSSION THREAD

**Discussion by stefer** [mburumaxwell/paklo#1520] [Q&A] [answered]
I tried to abandon a pull request because the updated package had issues.

The next day, depandabot tried to create the PR again and failed with this error message:
Unable to create pull request 'Bump NUnit from 4.2.2 to 4.3.0' as source branch 'dependabot/nuget/master/NUnit-4.3.0' already exists; Delete the existing branch and try again.

I thought abandoning pull requests would stop dependabot from creating them again?
If not, how can I ignore a package update?

I am using task version 2 and grouping, but grouping is only configured for other packages.

**Comment by rhyskoedijk** [comment, marked as answer]
@stefer currently this DevOps implementation of Dependabot does not automatically delete the source branch if you manually abandon the PR, nor will it automatically add the package to the ignore list.

The error you received is likely because the branch Dependabot created was not deleted after you abandoned the PR.
To ignore a package update, you must [add an ignore rule to your `dependabot.yml` config](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#ignore).

If you would like to see these features added, please raise an issue for them.
